src/macaron/repo_finder/repo_finder_java.py

```diff
@@ -0,0 +1,323 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I believe the implementation of methods inside `JavaRepoFinder` didn't change much from the previous state and you only move it into a class?

Reply:
This is true.